### ISSUE TITLE
FFTDeltaAtomProjection: another atom projection method using delta function approach

### ISSUE THREAD

**Issue by geoffwoollard** [michael-0brien/cryojax#477]
Inspired by comments Steve Ludtke made at the summer 2025 Flatiron meeting.

Does not require non-uniform fft.

**Comment by geoffwoollard** [comment]
I'm seeing some promising scaling.

- [ ] Pass in kernel hyperparams at integration init (kernel size, sigma)
- [ ] Allow for per atom amplitudes in upstream api (functionality already using this with hard coded all ones this)
- [ ] Match scaling (pixel size etc)
- [ ] Do numerical tests (perhaps on `FFTAtomProjection` and `GaussianMixtureProjection` where coordinates are pre-rounded to integer values)

<img width="2444" height="3566" alt="Image" src="https://github.com/user-attachments/assets/43783719-ce2f-434c-85f5-3f880864a463" />

<img width="1266" height="1166" alt="Image" src="https://github.com/user-attachments/assets/6833df15-c797-4fbc-a07b-dc6e3015c93a" />

**Comment by geoffwoollard** [comment]
> Can you elaborate with a minimal code example in this thread?

See the branch: https://github.com/michael-0brien/cryojax/pull/478

**Comment by michael-0brien** [comment]
> > Can you elaborate with a minimal code example in this thread?
> 
> See the branch: [#478](https://github.com/michael-0brien/cryojax/pull/478)

Reading through the method, this is a naive implementation of a non-uniform FFT.

In order for a new method to be accepted, it will need to be a similar level of generality and numerical accuracy as the current method. I don’t think this meets this criteria and seems simple enough to implement downstream. E.g.

```python

class AtomVolume(cxs.AbstractVolumeRepresentation):

    atom_positions: jax.Array
    gaussian_width: jax.Array


class DeltaIntegrator(cxs.AbstractVolumeIntegrator):

    n_trunc: int

    def integrate(volume_representation: AtomVolume, image_config: cxs.BasicImageConfig, outputs_real_space: bool = True):
        …
```

**Comment by geoffwoollard** [comment]
Curious what you are seeing in benchmarking with your fi-nufft install. Can you run `docs/examples/scripts/benchmark_projection_method_tradeoff.py` and share the plot?

**Comment by michael-0brien** [comment]
Sure can do that. FYI if you want to see better performance from the FINUFFT approach, you’ll have to tune the jax-finufft algorithm. Your current approach is optimized for numerical accuracy, so I don’t think the benchmark is fair. I would 1) Increase `eps` to be much larger, maybe 1e-3, and 2) Look into setting `opts` with `upsamplefac = 1` (see jax-finufft for details). By default, I believe FINUFFT automatically tries to figure out if it should compute on a more fine grid.  

I also noticed your GaussianMixtureVolume is using the tabulated 5 gaussian parameters while the other methods are not; for a fair comparison I would change this to 1 gaussian per atom.
